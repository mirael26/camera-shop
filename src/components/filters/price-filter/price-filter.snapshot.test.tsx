import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import PriceFilter from './price-filter';

test('Price-filter component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<PriceFilter/>);
  expect(asFragment()).toMatchSnapshot();
});
