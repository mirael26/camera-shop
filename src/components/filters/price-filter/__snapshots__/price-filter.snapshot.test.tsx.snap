// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price-filter component renders correctly 1`] = `
<DocumentFragment>
  <fieldset
    class="catalog-filter__block"
  >
    <legend
      class="title title--h5"
    >
      Цена, ₽
    </legend>
    <div
      class="catalog-filter__price-range"
    >
      <div
        class="custom-input"
      >
        <label>
          <input
            data-testid="price-input"
            name="price"
            placeholder="от"
            type="number"
            value=""
          />
        </label>
      </div>
      <div
        class="custom-input"
      >
        <label>
          <input
            data-testid="price-input"
            name="priceUp"
            placeholder="до"
            type="number"
            value=""
          />
        </label>
      </div>
    </div>
  </fieldset>
</DocumentFragment>
`;
